src/core/Jalasoft.TeamUp.Projects.Core/InvitationsService.cs

```diff
@@ -37,6 +40,20 @@ public Invitation UpdateInvitation(Invitation invitation)
         {
             InvitationValidator validator = new InvitationValidator();
             validator.ValidateAndThrow(invitation);
+            var invitationStatus = InvitationStatus.Accepted;
+            if (invitation.Status == invitationStatus.ToString())
```

Comment:
what will happen in this case?
if the invitation is already `Accepted`, if I make the PATCH of the invitation updating another field that is not the status, ex. invitation title, are we going to update again the project?

Reply:
well maybe is not a case that will exist because once the invitation is accepted the UI won't make the request again but it's a case that we need to take care of in other scenarios, I'd say it's a nice to have at this point

Reply:
Solved, Now we can´t add the same resume in the projects Member List